<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorForm</class>
 <widget class="QWidget" name="InspectorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="targetLabel">
           <property name="text">
            <string>Message Targets</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="targetCheckButton">
           <property name="text">
            <string>Check All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="targetUncheckButton">
           <property name="text">
            <string>Uncheck All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="messageTargetList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Message Types</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="typeCheckButton">
           <property name="text">
            <string>Check All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="typeUncheckButton">
           <property name="text">
            <string>Uncheck All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="messageTypeList"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear Stored Messages</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidgetList">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
